Makefile.abs.mk

```diff
@@ -30,8 +30,7 @@ release_ver_to_code:
 	$(call check_defined, TAG)
 	echo "build_ver = \"${TAG}\"\n" > app_build_suite/version.py
 	$(eval IMG_VER := ${TAG})
-	cp dabs.sh dabs.sh.back
-	bash -c 'sed -i "" "s/latest/$${TAG#v}/" dabs.sh'
+	bash -c 'sed -i.back "s/latest/$${TAG#v}/" dabs.sh'
```

Comment:
`-i` supports an optional suffix to create a temp file which we were doing already.

This should allow the Makefile to support both OSes.

See https://stackoverflow.com/questions/16745988/sed-command-with-i-option-in-place-editing-works-fine-on-ubuntu-but-not-mac/44864004#44864004 for more details